src/Console/Command/SelfUpdateCommand.php

```diff
@@ -43,7 +43,8 @@ protected function configure()
                 )
             )
             ->setDescription('Update php-cs-fixer.phar to the latest stable version.')
-            ->setHelp(<<<'EOT'
+            ->setHelp(
+                <<<'EOT'
```

Comment:
I'm not sure this is correct. It's not really a multi-line arguments IMHO, we only have one.